salty-747/html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/B747_8_EICAS.js

```diff
@@ -3,43 +3,53 @@ class B747_8_EICAS extends Airliners.BaseEICAS {
         super(...arguments);
         this.engines = new Array();
     }
+
     Init() {
         super.Init();
         for (let i = 0; i < Simplane.getEngineCount(); i++) {
             this.engines.push(new B747_8_Engine());
         }
         this.currentPage = "B747_8_EICAS_fuel";
     }
+
     reboot() {
         super.reboot();
         if (this.warnings)
             this.warnings.reset();
         if (this.annunciations)
             this.annunciations.reset();
     }
+
     onEvent(_event) {
         super.onEvent(_event);
         if (this.currentPage !== _event) {
             this.currentPage = _event;
         } else {
+            // if the event is happening on the same page, show blank for about half a second to make it look like the screen blinks before updating
```

Comment:
this actually shows blank until the page is changed again, not just for half a second